<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AboutOrderWindow</class>
 <widget class="QWidget" name="AboutOrderWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1110</width>
    <height>567</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1110</width>
    <height>567</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1110</width>
    <height>567</height>
   </size>
  </property>
  <property name="palette">
   <palette>
    <active/>
    <inactive/>
    <disabled/>
   </palette>
  </property>
  <property name="font">
   <font>
    <family>Inter</family>
    <pointsize>16</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="about_order_label">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>20</y>
     <width>411</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Inter</family>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Подробности о заказе</string>
   </property>
  </widget>
  <widget class="QLabel" name="status_order_label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Статус заказа</string>
   </property>
  </widget>
  <widget class="QLabel" name="date_start_order_label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>140</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Дата заказа</string>
   </property>
  </widget>
  <widget class="QLabel" name="name_service_label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>190</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Услуга</string>
   </property>
  </widget>
  <widget class="QLabel" name="price_service_label">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>90</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Стоимость</string>
   </property>
  </widget>
  <widget class="QLabel" name="type_machine_label">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>140</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Тип станка</string>
   </property>
  </widget>
  <widget class="QLabel" name="brand_machine_label">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>190</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Бренд станка</string>
   </property>
  </widget>
  <widget class="QLabel" name="description_order_label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>240</y>
     <width>421</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Описание проделанной работы</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="status_order_line_edit">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>90</y>
     <width>300</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="date_start_order_line_edit">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>140</y>
     <width>300</width>
     <height>31</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="name_service_line_edit">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>190</y>
     <width>300</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="brand_machine_line_edit">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>190</y>
     <width>300</width>
     <height>31</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="type_machine_line_edit">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>140</y>
     <width>300</width>
     <height>31</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="price_service_line_edit">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>90</y>
     <width>300</width>
     <height>31</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="description_order_text_edit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>280</y>
     <width>1031</width>
     <height>191</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="about_machine_button">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>495</y>
     <width>341</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Inter</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Подробности о станке</string>
   </property>
  </widget>
  <widget class="QPushButton" name="go_area_button">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>495</y>
     <width>341</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Inter</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Личный кабинет</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="date_end_line_edit">
   <property name="geometry">
    <rect>
     <x>809</x>
     <y>240</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="date_end_label">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>240</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Дата завершения</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
